data/virt_autotest/host_unattended_installation_files/autoyast_opensuse_kvm_sshd.xml

```diff
@@ -53,13 +53,19 @@
       <xen_append>console=hvc0,115200 console=tty loglevel=5</xen_append>
     </global>
   </bootloader>
-  <partitioning t="list">
+  <partitioning config:type="list">
     <drive>
       <device>/dev/sda</device>
       <initialize t="boolean">true</initialize>
       <use>all</use>
       <enable_snapshots t="boolean">false</enable_snapshots>
     </drive>
+    <drive>
+      <device>/dev/sdb</device>
+      <initialize t="boolean">true</initialize>
+      <use>all</use>
+      <enable_snapshots t="boolean">false</enable_snapshots>
+    </drive>
```

Comment:
I saw there has been some successful builds with this fix.  If adding the 2nd disk in autoyast profile could resolve the long exist failure, we can take it as the final fix because it saves effort.

But the limitation of this change is that these autoyast profile could be only used for machines with 2 and more hard disks. Machines with single disk will fail to install due to missing sdb, fg. http://10.67.129.27/tests/76#step/ipxe_install/2

It is not the best solution, but it can be acceptable as we need these tests go into TW job group quickly. So how about we change the name of this file to like 'autoyast_opensuse_kvm_two_disk.xml' to flag that it is limited to machines with 2 disks. We change the iPXE bootscripts in O3 to call this 'two_disk.xm', then the two tests on O3 will work as usual.

Meanwhile we create another 'autoyast_opensuse_kvm_one_disk.xml' on the same directory.  For installing TW outside O3, We can set `AUTOYAST=xxx` in test suites to specify which profiles to be used, two-disk or one-disk, based on your own worker.

Reply:
I kind of like this way and approach and thought process because other solutions tried earlier had one or more problems in other areas.

Reply:
modified the file names 
new file:   autoyast_opensuse_kvm_one_disk_sshd.xml
renamed:    autoyast_opensuse_kvm_sshd.xml -> autoyast_opensuse_kvm_two_disk_sshd.xml

Reply:
Julie, could you please modify it as I do not have edit permissions
Test Suites -> virt-guest-installation-kvm -> Settings -> AUTOYAST=xxx from "autoyast_opensuse_kvm_sshd.xml" to autoyast_opensuse_kvm_two_disk_sshd.xml

![twodisk profile picked](https://github.com/os-autoinst/os-autoinst-distri-opensuse/assets/46835794/203dfbe6-7b8f-4767-8a98-f1aefaf13b3b)
![2disk_selected](https://github.com/os-autoinst/os-autoinst-distri-opensuse/assets/46835794/8ee16e56-aba1-407a-b820-d41b19814038)

Reply:
Verification Run after file renamed: [KVM_rebel5](https://openqa.opensuse.org/tests/3544979)

Reply:
sure, thanks!

Reply:
renamed:    autoyast_opensuse_kvm_one_disk_sshd.xml -> autoyast_opensuse_kvm_one_disk.xml
renamed:    autoyast_opensuse_kvm_two_disk_sshd.xml -> autoyast_opensuse_kvm_two_disk.xml

Reply:
renamed:    autoyast_opensuse_xen_sshd.xml -> autoyast_opensuse_xen_two_disk.xml